rasa/__main__.py

```diff
@@ -64,7 +66,17 @@ def create_argument_parser() -> argparse.ArgumentParser:
 
 
 def print_version() -> None:
-    print("Rasa", version.__version__)
+    print(f"Rasa Version     : {version.__version__}")
+    py_stuff, os_stuff = sys.version.split("\n")
+    print(f"Python Version   : {py_stuff}")
+    print(f"Operating System : {os_stuff}")
+    print(f"Python Path      : {sys.path[0]}")
+    try:
+        import rasa_sdk
+
+        print(f"Rasa SDK Version : {rasa_sdk.__version__}")
+    except ModuleNotFoundError:
+        pass
```

Comment:
Does this fail if rasa sdk is not found like it did for rasa x? (I would expect this code to work 🤔 )

Reply:
I think rasa depends on Rasa SDK internally. On import independantly of the command that I am adding behavior to things will fail if it is not around. It is because by code like [this line](https://github.com/RasaHQ/rasa/blob/master/rasa/cli/arguments/run.py#L15).

Reply:
got it, makes sense -- rasa sdk is indeed a dependency of rasa. Then we shouldn't have an import try/except around it, no?

Reply:
Fixed. Will still include rasa-x just in case.